server/index.js

```diff
@@ -72,6 +72,11 @@ module.exports = (_config = {}) => {
 
   // get bpanel config
   const bpanelConfig = new Config('bpanel');
+
+  // inject any custom configs passed
```

Comment:
What happens in an empty `_config` object is passed? I feel like it should cause `bpanelConfig.inject` to do nothing, so you wouldn't need to check to make sure the object was populated

Reply:
yep, good call.

Reply:
Updated